src/api/app/services/scm_status_reporter.rb

```diff
@@ -13,18 +13,26 @@ def call
     if github?
       github_client = Octokit::Client.new(access_token: @scm_token, api_endpoint: @event_subscription_payload[:api_endpoint])
       # https://docs.github.com/en/rest/reference/repos#create-a-commit-status
-      github_client.create_status(@event_subscription_payload[:target_repository_full_name],
-                                  @event_subscription_payload[:commit_sha],
-                                  @state,
-                                  status_options)
+      begin
+        github_client.create_status(@event_subscription_payload[:target_repository_full_name],
+                                    @event_subscription_payload[:commit_sha],
+                                    @state,
+                                    status_options)
+      rescue Octokit::NotFound, Octokit::InternalServerError => e
+        Airbrake.notify(e)
```

Comment:
This makes very little sense as *exactly* this will happen automatically right?

Reply:
Do you mean `errbit` will be informed automatically?

Reply:
Yes, whenever an exception is raised without being rescued, Errbit is notified.

Reply:
Thanks for the correction. It's updated.